starter-code/app.js

```diff
@@ -1027,20 +1028,72 @@ var abbeyRoadRecords = [{song: "Land of Hope and Glory",
                       year: 2016}];
 
 
-
+//4.1
 
 var novemberArtists = function (abbeyRoadRecords) {
   var justNovember = abbeyRoadRecords.filter(function(element) {
     return element.month === 11;
     })
-
+  
   var myArtists = justNovember.map(function(element) {
     return element.artist;
-
+  
   })
-
+  
   return _.union(myArtists);
-
+  
 }
 
 console.log(novemberArtists(abbeyRoadRecords));
+  
+
+//4.2
+var bestArtist = function (abbeyRoadRecords) {
+  
+  
+};
+
+//4.3
+
+var lastBeatlesSong = function (abbeyRoadRecords) {
+    var justBeatles = abbeyRoadRecords.filter(function(element) {
+    return element.artist === "The Beatles";
+    })
+  
+  justBeatles.sort(function(song1,song2) {
+  return song1.year - song2.year;
+  })
+  
+  return _.head(justBeatles.reverse());
+  
+};
+
+console.log(lastBeatlesSong(abbeyRoadRecords));
+
+
+//4.4 
+
+var sixtiesSong = function (abbeyRoadRecords) {
+  var justSixties = abbeyRoadRecords.filter(function(element) {
+    return element.year === 1969;
+    })
+  
+  justSixties.sort(function (song1,song2) {
+    return song1.month > song2.month;
+  })
+  
+  return _.last(justSixties);
+ 
+};
+
+
```

Comment:
Good job! On the last exercise you could've used the _filter and _sort methods from lodash too